Stytch/DemoApps/StytchDemo/Shared/Info.plist

```diff
@@ -10,7 +10,7 @@
 			<key>CFBundleTypeRole</key>
 			<string>Editor</string>
 			<key>CFBundleURLName</key>
-			<string>com.stytch.StytchDemo</string>
+			<string>com.stytch.Stytch</string>
```

Comment:
I feel like maybe the bundle ID should stay as `StytchDemo`?

Reply:
fixed